timescaledb/alpine/3.9/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2019 Crave.io Inc. All rights reserved
+
+# Take base of postgres
+FROM postgres:alpine
+
+# Apply accupara docker image standards base
+FROM accupara/alpine:3.9
```

Comment:
Only one FROM works, doesn't it?

Reply:
2 works. it will merge the two containers

Reply:
Oh? Looks like new functionality. Ok.